indexing-service/src/main/java/org/apache/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTask.java

```diff
@@ -550,7 +546,9 @@ public Response getUnparseableEvents(
   )
   {
     IndexTaskUtils.datasourceAuthorizationCheck(req, Action.READ, getDataSource(), authorizerMapper);
-    List<String> events = IndexTaskUtils.getMessagesFromSavedParseExceptions(savedParseExceptions);
+    List<String> events = IndexTaskUtils.getMessagesFromSavedParseExceptions(
+        parseExceptionHandler.getSavedParseExceptions()
```

Comment:
parseExceptionHandler can be null if the `task.run()` hasn't been called. This is probably unlikely?

Reply:
This is fine. Any APIs cannot be called until `chatHandler` is registered. The `chatHandler` is registered in `run()` after `parseExceptionHandler` is created.